includes/get-shortcodes.php

```diff
@@ -1,47 +1,67 @@
 <?php
-if ( !defined( 'ABSPATH' ) ) {
-    die( "Sorry, you are not allowed to access this page directly." );
+/**
+ * Get Global Shortcodes
+ * 
+ * @package Shortcode_Lister
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	die( 'Sorry, you are not allowed to access this page directly.' );
 }
 add_filter( 'media_buttons', 'shortcode_lister_menu', 15 );
```

Comment:
@richdacuban 'media_buttons' is actually an action. so this should be `add_action` instead. Then you won't need to return null. It would be void instead.

https://developer.wordpress.org/reference/hooks/media_buttons/